nuxt.config.js

```diff
@@ -104,8 +104,16 @@ module.exports = {
         fallbackLocale: defaultLocale,
       }
     }],
+    '@nuxtjs/sitemap'
   ],
 
+  sitemap: {
+    routes () {
```

Comment:
this is the same object as under the `generate` property, so ideally, they share that code, rather than having two different versions.